src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/hosts.component.ts

```diff
@@ -181,8 +226,67 @@ export class HostsComponent extends ListWithDetails implements OnInit {
     });
   }
 
-  getDisable(action: 'create' | 'edit' | 'delete', selection: CdTableSelection): boolean | string {
-    if (action === 'delete' || action === 'edit') {
+  hostMaintenance() {
+    this.isExecuting = true;
+    const host = this.selection.first();
+    if (host['status'] !== 'maintenance') {
+      this.hostService.update(host['hostname'], false, [], true).subscribe(
+        () => {
+          this.isExecuting = false;
+          this.notificationService.show(
+            NotificationType.success,
+            $localize`"${host.hostname}" moved to maintenance`
+          );
+          this.table.refreshBtn();
+        },
+        (error) => {
+          this.isExecuting = false;
+          this.errorMessage = error.error['detail'].split(/\n/);
+          error.preventDefault();
+          if (
+            error.error['detail'].includes('WARNING') &&
+            !error.error['detail'].includes('It is NOT safe to stop') &&
```

Comment:
We should check with orch. team if these conditions are the appropriate ones.

Reply:
Reviewing with @nizamial09 , we have conclude that:
Only is useful to show the "force" modal dialog box, if all the messages coming from the orchestrator output are "WARNINGS". In this case all these warnings can by bypassed using the force parameter.
If in the orch output we have "ALERT" or "It is not safe to stop"  then it has no sense to present any "force" dialog. because to stop the daemons in the host is going to "break" the cluster. 

If the force flag is not going to help... maybe it would be nice to to show a informative dialog box (instead of the force dialog) explaining a little bit more the situation ... because this situation could change if the user makes the cluster more robust providing more daemons redundancy  (mons, mgr, osds).

Reply:
@jmolmo Initially I showed a dialog box for errors like `It is NOT SAFE to stop` like this:
![108093159-b6800c00-707d-11eb-9ea7-f4419aa2a306](https://user-images.githubusercontent.com/71764184/110317774-9adfa400-8032-11eb-8ce3-168a12980eac.png)

But @epuertat told in this comment https://github.com/ceph/ceph/pull/39226#pullrequestreview-591248945 that it would be useful to show the modal for the user only when there is an action is present (like Continue) else the user can only read and  cancel the box. That's why I went with showing just the notification for cases like this.